rohmu/util.py

```diff
@@ -18,7 +18,7 @@
 import platform
 import types
 
-LOG = logging.getLogger("rohmu.util")
+LOG = logging.getLogger(__name__)
```

Comment:
nit (non-blocking): this change is unrelated to pydantic, should probably go into separate commit